zerver/migrations/0501_mark_introduce_zulip_view_modals_as_read.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 4.2.10 on 2024-03-11 04:43
+
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+from django.db.models import Q
+
+
+def mark_introduce_zulip_view_modals_as_read(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
+    UserProfile = apps.get_model("zerver", "UserProfile")
+    OnboardingStep = apps.get_model("zerver", "OnboardingStep")
+
+    user_profiles = UserProfile.objects.filter(is_bot=False)
+
+    onboarding_steps = []
+    for user_profile in user_profiles:
+        onboarding_steps.append(
+            OnboardingStep(user=user_profile, onboarding_step="intro_inbox_view_modal")
+        )
+        onboarding_steps.append(
+            OnboardingStep(user=user_profile, onboarding_step="intro_recent_view_modal")
+        )
+
+    OnboardingStep.objects.bulk_create(onboarding_steps, batch_size=10000)
```

Comment:
[Answered on CZO](https://chat.zulip.org/#narrow/stream/3-backend/topic/onboarding_steps.20migration/near/1759136), but this would be very easy to write as a `INSERT INTO ... SELECT` statement, and would be faster and less memory-intensive.